perf/statz/distributions.go

```diff
@@ -16,6 +16,9 @@ type Distribution struct {
 // LatencyDistribution is a basic latency distribution.
 var LatencyDistribution = DistributionFromBounds(0, 5, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000)
 
+// ConnectionTimeDistribution is a basic distribution for connection times in seconds.
+var ConnectionTimeDistribution = DistributionFromBounds(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
```

Comment:
Nice TIL about OC distributions. From what I can tell, this might be the first usage of that metric type we'll have in GCP, as that `LatencyDistribution` above is only used in example code. I'm curious how it'll appear in GCP or what it will be like to set up alerting policies against it.

Reply:
seconds is too coarse - can we have more buckets? ideally granularity of 250 or 500ms. If that's too many buckets, I'd be ok removing some of the buckets in the 7/8/9s range.